src/univariate/continuous/skewedexponentialpower.jl

```diff
@@ -0,0 +1,118 @@
+
+```julia
+SkewedExponentialPower()            # SEPD with shape 2, scale 1, location 0, and skewness 0.5 (the standard normal distribution)
+SkewedExponentialPower(μ, σ, p, α)  # SEPD with location μ, scale σ, shape p, and skewness α
+SkewedExponentialPower(μ, σ, p)     # SEPD with location μ, scale σ, shape p, and skewness 0.5 (the exponential power distribution)
+SkewedExponentialPower(μ, σ)        # SEPD with location μ, scale σ, shape 2, and skewness 0.5 (the normal distribution)
+SkewedExponentialPower(μ)           # SEPD with location μ, scale 1, shape 2, and skewness 0.5 (the normal distribution)
+
+params(d)       # Get the parameters, i.e. (μ, σ, p, α)
+shape(d)        # Get the shape parameter, i.e. p
+location(d)     # Get the location parameter, i.e. μ
+scale(d)        # Get the scale parameter, i.e. σ
+```
+"""
+struct SkewedExponentialPower{T <: Real} <: ContinuousUnivariateDistribution
+    μ::T
+    σ::T
+    p::T
+    α::T
+    SkewedExponentialPower{T}(μ::T, σ::T, p::T, α::T) where {T} = new{T}(μ, σ, p, α)
+end
+
+function SkewedExponentialPower(µ::T, σ::T, p::T, α::T; check_args=true) where {T <: Real}
+    if check_args
+        @check_args(SkewedExponentialPower, σ > zero(σ))
+        @check_args(SkewedExponentialPower, p > zero(p))
+        @check_args(SkewedExponentialPower, zero(α) < α < one(α))
+    end
+    return SkewedExponentialPower{T}(µ, σ, p, α)
+end
+
+function SkewedExponentialPower(μ::Real=0, σ::Real=1, p::Real=2, α::Real=1//2; kwargs...)
+    return SkewedExponentialPower(promote(μ, σ, p, α)...; kwargs...)
+end
+
+@distr_support SkewedExponentialPower -Inf Inf
+
+### Conversions
+convert(::Type{SkewedExponentialPower{T}}, μ::S, σ::S, p::S, α::S) where {T <: Real, S <: Real} = SkewedExponentialPower(T(μ), T(σ), T(p), T(α))
+convert(::Type{SkewedExponentialPower{T}}, d::SkewedExponentialPower{S}) where {T <: Real, S <: Real} = SkewedExponentialPower(T(d.μ), T(d.σ), T(d.p), T(d.α), check_args=false)
+convert(::Type{SkewedExponentialPower{T}}, d::SkewedExponentialPower{T}) where {T<:Real} = d
+
+### Parameters
+@inline partype(d::SkewedExponentialPower{T}) where {T<:Real} = T
+
+params(d::SkewedExponentialPower) = (d.μ, d.σ, d.p, d.α)
+location(d::SkewedExponentialPower) = d.μ
+shape(d::SkewedExponentialPower) = d.p
+scale(d::SkewedExponentialPower) = d.σ
+
+### Statistics
+
+#Calculates the kth central moment of the SEPD on the log-scale
+function m_k(d::SkewedExponentialPower, k::Integer)
+    _, σ, p, α = params(d)
+    return (k*log(2) + k*inv(p) * log(p) + k*log(σ) + loggamma((1+k)*inv(p)) -
+        loggamma(inv(p))) + log((-1)^k*α^(1+k) + (1-α)^(1+k))
+end
+
+mean(d::SkewedExponentialPower) = d.α == 0.5 ? d.μ : exp(m_k(d, 1)) + d.μ
+mode(d::SkewedExponentialPower) =  mean(d)
+var(d::SkewedExponentialPower) = exp(m_k(d, 2)) - exp(2*m_k(d, 1))
+skewness(d::SkewedExponentialPower) = d.α == 0.5 ? 0 : exp(m_k(d, 3)) / (√(v(d)))^3
```

Comment:
Similarly, for certain types of parameters `skewness` is type unstable. Either compute the second branch and return a zero of its type or use a heuristic such as
```suggestion
skewness(d::SkewedExponentialPower) = d.α == 1//2 ? float(zero(partype(d))) : exp(m_k(d, 3)) / (√(v(d)))^3
```
I assume that `v` is not defined? It is not covered by the tests currently though since you always use `alpha = 0.5`. Can you fix this?